test/export/StructureDefinition.ResourceExporter.test.ts

```diff
@@ -262,19 +263,98 @@ describe('ResourceExporter', () => {
 
     const exported = exporter.export().resources[0];
 
+    expect(exported.name).toBe('MyTestResource');
+    expect(exported.id).toBe('MyResource');
+    expect(exported.type).toBe('MyResource');
+    expect(exported.baseDefinition).toBe('http://hl7.org/fhir/StructureDefinition/DomainResource');
+    expect(exported.elements).toHaveLength(12);
+
```

Comment:
I'm sorry to keep doing this, but I'd really like these lines to show up without any differences. To do that, you need to change two things:
- The comment is missing on the last expect statement. We need to keep this because it provides context about why we are expecting to have a length of `12`. It's a pretty arbitrary looking number, so we don't want to lose that context.
- These lines should get added _underneath_ the expect statement checking each log statement. Looking at the current branch as it is now, the lines should get added after line 272. This change is technically not required since it doesn't make any difference in the order of the expect statements, but I think it's good to avoid making unnecessary changes in the tests. Before pushing this change, if you want to check that the lines are actually the same as they are on master, you can run `git diff master test/export/StructureDefinition.ResourceExporter.test.ts`. When you run that, the only differences you should see are the additions of the import and the three tests you added. You should no longer see that any lines were removed.

Reply:
Done!